src/packages/console/src/App.tsx

```diff
@@ -107,6 +107,9 @@ const theme = buildTheme({
       'border-color': '$colors.backgroundDark25',
       'border-radius': '0',
     },
+    unpadded: {
```

Comment:
not a great name because there is padding still. `paddedNarrow` is probably better

Reply:
sorry, I mean to have no padding here, forgot to save the file.